ci/rholang-more-tests-main.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+export PATH=$PATH:$(pwd -P)/ci
+source header.sh
+
+${PROJECT_ROOT_DIR}/ci/build-ubuntu-rosette.sh
```

Comment:
I think @nashef's suggestion was to use Nix to build the binary for running the `rholang_more_tests` Travis `SUBPROJECT`.

See `build-subprojects.sh` for how I'm doing this with the `rosette` Travis `SUBPROJECT`